### DISCUSSION TITLE
When doing activation flow / reset password flow, is there a requirement to use the same browser ?

### DISCUSSION THREAD

**Discussion by mderazon** [ory/kratos#1761] [Q&A] [answered]
For example, in the account activation flow:
https://www.ory.sh/kratos/docs/self-service/flows/verify-email-account-activation
Is there a requirement for the user to start and finish the flow on the same browser ? (basically click on the link in the email in the same browser that requested the account activation)

Thanks 👍🏼

**Comment by aeneasr** [comment, marked as answer]
Nope, can be done in separate browsers!

**Reply by mderazon** [reply]
Thanks, I was trying to think if that poses a security risk in some way and couldn't get to a conclusion.
One use case I also thought of is multiple people sharing an email address (common in orgs) - What if some other employee clicked on the link...
